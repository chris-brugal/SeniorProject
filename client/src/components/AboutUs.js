import React from "react";
import { Card, Grid, CardHeader, Typography } from "@mui/material";
import "../styles/LandingPage.css";
import { Box } from "@mui/system";

const AboutUs = () => {
  return (
    <div id="about" className="About Us">
      <Typography
        className="main-text"
        sx={{
          fontWeight: 525,
          paddingLeft: {
            xs: "0px",
            md: "30px",
          },
          paddingTop: {
            xs: "30px",
            md: "0px",
          },
        }}
        variant="h2"
      >
        About Spotify Central
      </Typography>
      <Grid
        container
        spacing={3}
        alignItems="center"
        justifyContent="center"
        style={{ minHeight: "95vh" }}
      >
        <Grid item xs={12} md={3} container justify="center">
          <Card className="card-size">
            <Box
              sx={{
                display: "flex",
                justifyContent: "center",
                p: {
                  xs: "10px",
                },
              }}
            >
              <img src="/images/graph.png" className="icon-img" alt="graph" />
            </Box>
            <CardHeader
              title={"Statistics"}
              subheader={
                "Be able to see your statics such as your top artists and albums!"
              }
              className="card-title"
            />
          </Card>
        </Grid>
        <Grid item xs={12} md={3} container justify="center">
          <Card className="card-size">
            <Box
              sx={{
                display: "flex",
                justifyContent: "center",
                p: {
                  xs: "10px",
                },
              }}
            >
              <img src="/images/trail.png" className="icon-img" alt="trail" />
            </Box>
            <CardHeader
              title={"Generate"}
              subheader={
                "Generate perfect length playlists for when you are on the road!"
              }
              className="card-title"
            />
          </Card>
        </Grid>
        <Grid item xs={12} md={3} container justify="center">
          <Card className="card-size">
            <Box
              sx={{
                display: "flex",
                justifyContent: "center",
                p: {
                  xs: "10px",
                },
              }}
            >
              <img src="/images/album.png" className="icon-img" alt="album" />
            </Box>
            <CardHeader
              title={"Personalize"}
              subheader={
                "Be able to customize and edit your playlists much more easily!"
              }
              className="card-title"
            />
          </Card>
        </Grid>
      </Grid>
    </div>
  );
};

export default AboutUs;
