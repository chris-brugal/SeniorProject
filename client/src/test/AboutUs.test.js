import React from 'react';
import AboutUs from '../components/AboutUs';
import renderer from "react-test-renderer";

describe("snapshot testing", () => {

test("snapshot testing for AboutUs component", () => {
    const comp = renderer.create(<AboutUs />).toJSON()
    expect(comp).toMatchSnapshot()
})
})