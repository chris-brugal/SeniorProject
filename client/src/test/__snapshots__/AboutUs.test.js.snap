// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing snapshot testing for AboutUs component 1`] = `
<div
  className="About Us"
  id="about"
>
  <h2
    className="MuiTypography-root MuiTypography-h2 main-text css-lrs2p1-MuiTypography-root"
  >
    About Spotify Central
  </h2>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 css-1402ce4-MuiGrid-root"
    style={
      Object {
        "minHeight": "95vh",
      }
    }
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-3 css-1rymtov-MuiGrid-root"
      justify="center"
    >
      <div
        className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card-size css-bhp9pd-MuiPaper-root-MuiCard-root"
      >
        <div
          className="MuiBox-root css-scc2th"
        >
          <img
            alt="graph"
            className="icon-img"
            src="/images/graph.png"
          />
        </div>
        <div
          className="MuiCardHeader-root card-title css-185gdzj-MuiCardHeader-root"
        >
          <div
            className="MuiCardHeader-content css-1qbkelo-MuiCardHeader-content"
          >
            <span
              className="MuiTypography-root MuiTypography-h5 MuiCardHeader-title css-1qvr50w-MuiTypography-root"
            >
              Statistics
            </span>
            <span
              className="MuiTypography-root MuiTypography-body1 MuiCardHeader-subheader css-nrdprl-MuiTypography-root"
            >
              Be able to see your statics such as your top artists and albums!
            </span>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-3 css-1rymtov-MuiGrid-root"
      justify="center"
    >
      <div
        className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card-size css-bhp9pd-MuiPaper-root-MuiCard-root"
      >
        <div
          className="MuiBox-root css-scc2th"
        >
          <img
            alt="trail"
            className="icon-img"
            src="/images/trail.png"
          />
        </div>
        <div
          className="MuiCardHeader-root card-title css-185gdzj-MuiCardHeader-root"
        >
          <div
            className="MuiCardHeader-content css-1qbkelo-MuiCardHeader-content"
          >
            <span
              className="MuiTypography-root MuiTypography-h5 MuiCardHeader-title css-1qvr50w-MuiTypography-root"
            >
              Generate
            </span>
            <span
              className="MuiTypography-root MuiTypography-body1 MuiCardHeader-subheader css-nrdprl-MuiTypography-root"
            >
              Generate perfect length playlists for when you are on the road!
            </span>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-3 css-1rymtov-MuiGrid-root"
      justify="center"
    >
      <div
        className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card-size css-bhp9pd-MuiPaper-root-MuiCard-root"
      >
        <div
          className="MuiBox-root css-scc2th"
        >
          <img
            alt="album"
            className="icon-img"
            src="/images/album.png"
          />
        </div>
        <div
          className="MuiCardHeader-root card-title css-185gdzj-MuiCardHeader-root"
        >
          <div
            className="MuiCardHeader-content css-1qbkelo-MuiCardHeader-content"
          >
            <span
              className="MuiTypography-root MuiTypography-h5 MuiCardHeader-title css-1qvr50w-MuiTypography-root"
            >
              Personalize
            </span>
            <span
              className="MuiTypography-root MuiTypography-body1 MuiCardHeader-subheader css-nrdprl-MuiTypography-root"
            >
              Be able to customize and edit your playlists much more easily!
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
